packaging/suse/portusctl/lib/constants.rb

```diff
@@ -0,0 +1,5 @@
+# Some useful constants used by portus
+
+BUNDLER_BIN = "/srv/Portus/vendor/bundle/ruby/2.1.0/bin/bundler.ruby2.1"
+GEM_PATH    = "/srv/Portus/vendor/bundle/ruby/2.1.0/"
+HOSTNAME    = `hostname -f`.chomp
```

Comment:
if you want to share this with bash... couldn't we have a bash wiht

BUNDLER_BIN="......"
GEM_PATH="......."

so both requiring this file in ruby or in bash would work?